tests/tck/features/expression/Predicate.feature

```diff
@@ -49,6 +49,26 @@ Feature: Predicate
       | "Manu Ginobili"     | 95       |
       | "Tim Duncan"        | 95       |
 
+  Scenario: exists with dynamic map in MATCH
+    Given a graph with space named "nba"
+    When executing query:
+      """
+      MATCH(n:player) WHERE EXISTS(n['name'])
```

Comment:
Add a test case for only a part vertex contains property.

Reply:
ACK.

Reply:
please add case : match (a:player )-[b:like|:serve]->(c) where exists(b['likeness']) return a

Reply:
> please add case : match (a:player )-[b:like|:serve]->(c) where exists(b['likeness']) return a

This case has been added. Thanks for pointing that out.

Reply:
`match (v:player)-[]->(n) return exists(n["age"])`